tests/unit/assessment/test_pipelines.py

```diff
@@ -128,3 +242,34 @@ def test_pipeline_without_owners_should_have_empty_creator_name():
             failures="[]",
         )
     ]
+
+
+def test_pipeline_assessment_with_no_cluster_config(mocker):
+    sample_pipelines = [
+        PipelineStateInfo(
+            cluster_id=None,
+            creator_user_name="abcde.defgh@databricks.com",
+            latest_updates=None,
+            name="New DLT Pipeline",
+            pipeline_id="0112eae7-9d11-4b40-a2b8-6c83cb3c7407",
+            run_as_user_name="abcde.defgh@databricks.com",
+            state=PipelineState.IDLE,
+        )
+    ]
+
+    ws = create_autospec(WorkspaceClient)
```

Comment:
refactor to start using `workspace_client_mock`:

https://github.com/databrickslabs/ucx/blame/main/tests/unit/assessment/test_clusters.py#L22-L58

Reply:
replaced it with workspace_client_mock